Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM python:3.7
+
+RUN apt-get update
+RUN git clone https://github.com/alan-turing-institute/chance-water-distribution
```

Comment:
Would it be better to use COPY here? In order to save cloning the repo twice.

Reply:
So what I'm thinking here is that a user (e.g. someone who wants to run the app with their own network once further modifications are made to handle this) will be able to run the app with the docker commands only, i.e. not by cloning the repo

Reply:
I'm not sure I understand what you mean. I think if you want to add your own data, it might be easier to do something like
```Dockerfile
FROM chance-webapp

COPY my-network
```
or even using a docker volume pointing to your own data.

I suspect this will significantly slow down the automatic builds.

Reply:
Ah hang on, ignore my previous comment. I was getting confused about how Docker works. You're right, it does make sense to use COPY.